src/components/PresenceMap/readme.md

```diff
@@ -0,0 +1,183 @@
+            heading: 90,
+        },
+        {
+            position: {
+                lat: -35.850538,
+                lng: 151.1279,
+            },
+            image: "http://chart.googleapis.com/chart?chst=d_map_pin_letter&chld=xxx%7c5680FC%7c000000&.png",
+        },
+    ];
+
+    useEffect(() => {
+        setShowTraffic(showTrafficState);
+    }, [showTrafficState]);
+
+    useEffect(() => {
+        setShowTransit(showTransitState);
+    }, [showTransitState]);
+
+    useEffect(() => {
+        setZoom(zoomState);
+    }, [zoomState]);
+
+    useEffect(() => {
+        setCenter(centerState);
+    }, [centerState]);
+
+    const handleShowTraffic = () => {
+        setShowTraffic(!showTrafficState);
+    };
+
+    const handleShowTransit = () => {
+        setShowTransit(!showTransitState);
+    };
+
+    const handleCenterGeolocation = () => {
+        const geo_success = position => {
+            setCenter({
+                lat: position.coords.latitude,
+                lng: position.coords.longitude
+            });
+        };
+
+        const geo_error = () => {
+            alert("Sorry, no position available.");
+        };
+
+        const geo_options = {
+            enableHighAccuracy: true, 
+            maximumAge: 30000, 
+            timeout: 27000,
+        };
+
+        navigator.geolocation.getCurrentPosition(geo_success, geo_error, geo_options);
+    };
+
+    const handleMapStyle = value => {
+        setMapType(value);
+    };
+
+    return (
+        <PresenceMap
+            apiKey={LIBRARY_GOOGLE_MAPS_APIKEY}
+            objects={objects}
+            showTraffic={showTrafficState}
+            showTransit={showTransitState}
+            zoom={zoomState}
+            center={centerState}
+            type={mapTypeState.name}
+        >
+            <MenuControl
+                id="picklist-mapstyle-1"
+                style={{ float: "right" }}
+                onChange={value => handleMapStyle(value)}
+                value={mapTypeState}
+                label=""
+                hideLabel
+            >
+                <PicklistOption name="roadmap" label="ROADMAP" />
+                <PicklistOption name="satellite" label="SATELLITE" />
+                <PicklistOption name="hybrid" label="HYBRID" />
+                <PicklistOption name="terrain" label="TERRAIN" />
+            </MenuControl>
+            <RegularControl
+                className="rainbow-m-around_medium"
+                style={{ float: "right" }}
+                type="checkbox"
+                label="Show Traffic"
+                onClick={handleShowTraffic}
+                checked={showTrafficState} />
+            <RegularControl
+                className="rainbow-m-around_medium"
+                style={{ float: "right" }}
+                type="checkbox"
+                label="Show Transit"
+                onClick={handleShowTransit}
+                checked={showTransitState} />
+            <CircularControl
```

Comment:
Here we should use the ButtonIcon component which uses an interactive element like button, div is not meant to be clickable